src/dispatchers/mouseDispatcher.ts

```diff
@@ -191,7 +191,7 @@ export module Dispatchers {
      *
      * @return {Point}
      */
-    public lastMousePosition() {
+    public lastMousePosition(): Point {
```

Comment:
It's not really a bug from Typescript side, because how would Typescript know that this is a `Point` rather than an x,y object?

That said, I would much rather have Typescript better support implicit return types on types aliases